src/ViewLayer/DebugDisplay/Tab/TabUi/TabUi.cpp

```diff
@@ -26,6 +26,8 @@ TabUi::TabUi() :
     ui_->faultButton->setIconSize(QSize(33, 33));
     ui_->mpptButton->setIcon(QIcon(":/Resources/MPPTIcon.png"));
     ui_->mpptButton->setIconSize(QSize(38, 40));
+    ui_->auxBMSButton->setIcon(QIcon(":/Resources/GearIcon.png"));
```

Comment:
We don't have a graphic designer anymore so I used the Gear Icon that was already in the folder lol .